apps/api/src/scraper/scrapeURL/transformers/llmExtract.ts

```diff
@@ -259,7 +259,7 @@ export async function generateCompletions({
   markdown,
   previousWarning,
   isExtractEndpoint,
-  model = getModel("gpt-4o-mini", "openai"),
+  model = getModel("gpt-4o", "openai"),
```

Comment:
@mogery  Also, updated the model to gpt-4o, as mini doesn't work for recursive schema extraction, lmk if this is ok?

Reply:
Is there a way to detect recursive schemas and only use `gpt-4o` if they are detected? Also print a log if this happens so we can monitor

Reply:
yes, I've added model selection and logging function